scripts/skills/perks/perk_rf_fresh_and_furious.nut

```diff
@@ -26,31 +28,43 @@ this.perk_rf_fresh_and_furious <- ::inherit("scripts/skills/skill", {
 	{
 		local tooltip = this.skill.getTooltip();
 
-		tooltip.push({
-			id = 10,
-			type = "text",
-			icon = "ui/icons/special.png",
-			text = "The next skill used has its Action Point cost [color=" + ::Const.UI.Color.PositiveValue + "]halved[/color]"
-		});
+		if (this.m.RequiresRecover)
+		{
+			tooltip.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/warning.png",
+				text = ::MSU.Text.colorRed("Disabled until this character uses [Recover|Skill+recover]")
+			});
+		}
+		else
+		{
+			tooltip.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "The next non-free skill used has its Action Point cost [color=" + ::Const.UI.Color.PositiveValue + "]halved[/color]"
```

Comment:
From the gitlab POV it looks like you are touch all this text at once. At that point you might aswell fix the coloring here by using `::MSU.Text.colorGreen("halved")`